src/generator.cpp

```diff
@@ -1931,10 +1922,12 @@ void Generator::GeneratorImpl::addImplementationComputeRatesMethodCode(std::vect
             // NLA equation in case the rate is not on its own on either the LHS
             // or RHS of the equation.
 
+            auto variables = libcellml::analyserVariables(equation);
```

Comment:
Why are these function calls given the libcellml namespace? Does some compiler need them?